### PR TITLE
fix issue #6

### PR DESCRIPTION
**Pull request by liujiescut**
add debug function in simulator

### CODE REVIEW

File: commons/src/main/java/com/alibaba/weex/commons/util/OtherUtil.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ * @author Senyu, email: senyu.ylj@alibaba-inc.com
+ */
+public class OtherUtil {
+    /**
+     * 判断当前设备是否是模拟器。如果返回TRUE，则当前是模拟器，不是返回FALSE
+     */
+    public static boolean isEmulator(Context context) {
+        try {
+            TelephonyManager tm = (TelephonyManager) context
+                    .getSystemService(Context.TELEPHONY_SERVICE);
```
Comment (littleseven) [inline commons/src/main/java/com/alibaba/weex/commons/util/OtherUtil.java]: may need permission check on android 6+ 
<uses-permission android:name="android.permission.READ_PHONE_STATE"/>

File: playground/app/src/main/java/com/alibaba/weex/SimulatorDebugActivity.java [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    /**
+     * 初始化监听
+     */
+    private void initListener(){
+        mConnectDevToolButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                if(!mDebugHostEditText.getText().toString().
+                        matches("\\d{1,3}\\.\\d{1,3}\\.\\d{1,3}\\.\\d{1,3}")){
+                    Toast.makeText(SimulatorDebugActivity.this, "请输入有效服务器Ip", Toast.LENGTH_SHORT).show();
```
Comment (littleseven) [inline playground/app/src/main/java/com/alibaba/weex/SimulatorDebugActivity.java]: String constant write in values.xml would be better, that is convenient to add descriptor of multi-language.

File: commons/src/main/java/com/alibaba/weex/commons/util/OtherUtil.java (hunk 1 above)
Comment (liujiescut) [inline commons/src/main/java/com/alibaba/weex/commons/util/OtherUtil.java]: I have fixed this problem

File: playground/app/src/main/java/com/alibaba/weex/SimulatorDebugActivity.java (hunk 2 above)
Comment (liujiescut) [inline playground/app/src/main/java/com/alibaba/weex/SimulatorDebugActivity.java]: I have put the String constant in string.xml

**Pull request merged**